src/main/java/com/coronacarecard/service/impl/AWSS3ServiceImpl.java

```diff
@@ -0,0 +1,82 @@
+package com.coronacarecard.service.impl;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.*;
+import com.coronacarecard.exceptions.InternalException;
+import com.coronacarecard.service.AWSS3Service;
+import org.springframework.stereotype.Service;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Optional;
+
+@Service
+public class AWSS3ServiceImpl implements AWSS3Service {
+
+    private static final String AWS_ACCESS_KEY        = "AKIAQECDVEZ5V4CES3KO";
+    private static final String AWS_ACCESS_SECRET_KEY = "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";
+
+    @Override
+    public Bucket createBucket(String bucketName) throws InternalException {
+        // TODO Implement this method
+        return null;
+    }
+
+    @Override
+    public PutObjectResult uploadImage(String bucketName, String imageName, byte[] image, Optional<String> contentType)
+            throws InternalException {
+        PutObjectResult result = null;
+        AmazonS3Client  client = getAWSClient();
```

Comment:
Move this to constructor and inject it so it's initlize once as a part of depency inject look at SNSService